MSProject/ViewControllers/LibraryTableView/WBLibraryTableViewController.swift

```diff
@@ -47,6 +79,21 @@ class WBLibraryTableViewController: UIViewController {
         loadBooks()
     }
     
+    // MARK: - Search
+    func searchBarIsEmpty() -> Bool {
+        return searchController.searchBar.text?.isEmpty ?? true
+    }
+    
+    func filterContentForSearchText(_ searchText: String) {
+        isFiltering()
+        viewModel.filterBooks(with: searchText)
+        _view.bookTable.reloadData()
+    }
+    
+    func isFiltering() {
```

Comment:
this is confusing, i would expect this to return a value, not update the viewmodel.